### ISSUE TITLE
[grid-health] nightly-accessibility.yml unhealthy

### ISSUE THREAD

**Issue by tatteddev** [HoneyDrunkStudios/HoneyDrunk.Actions#135]
Grid health classified `nightly-accessibility.yml` in `HoneyDrunk.Actions` as **Missing**.

Latest run: none

Updated: 2026-07-13T06:37:05Z
